### DISCUSSION TITLE
Hi, my friends. I have a big problem. Please help.

### DISCUSSION THREAD

**Discussion by kjam99** [community/community#146919] [Programming Help] [answered]
### Select Topic Area

General

### Body

After changing the library for my Android Studio app from android.play:core to android.play:asset I have a problem with MainActivity / Import ( 35 ) Problem . I have no experience in this field, please help or if there is a video to solve the problem. Thank you all.

**Comment by mhaemnn** [comment, marked as answer]
The issue you're encountering likely stems from missing or incorrect imports or dependencies in your project after switching the library from androidx.play:core to androidx.play:asset.

1. Ensure Proper Dependency Installation

- Open your app-level build.gradle file and ensure you've added the correct dependency for androidx.play:asset. For example:
implementation 'androidx.play:asset:<version>'
Replace <version> with the latest version available [here](https://developer.android.com/jetpack/androidx/releases/play).

**Reply by Ownerbusinessjjfm** [reply]
That user is phishing others people phone stealing assets
